src/app/components/token/token.component.html

```diff
@@ -95,46 +131,84 @@
             </mat-menu>
         </ng-container>
     </div>
-    <div class="card__network">
-        <span
-            *ngIf="!allNetworksView; else all_networks"
-            matTooltip="{{ token.name }}
-                {{ token.address }}
-                (Click to copy)"
-            ngxClipboard
-            [cbContent]="token.address"
-        >
-            {{ token.symbol }}
-        </span>
+
+    <div class="card__network" fxLayout="row">
+        <div *ngIf="!isAllNetworksView; else all_networks">
+            <ng-container *ngIf="selectedToken.connected; else no_deposits">
+                <span
+                    *ngIf="
+                        selectedToken.connected?.sum_deposits
+                            | decimal: selectedToken.decimals as sumDeposits
+                    "
+                    [matTooltip]="sumDeposits"
+                    >{{ sumDeposits | displayDecimals: 3 }}</span
+                >
+            </ng-container>
+            <ng-template #no_deposits>
+                0
+            </ng-template>
+            <span
+                matTooltip="{{ selectedToken.name }}
+                    {{ selectedToken.address }}
+                    (Click to copy)"
+                ngxClipboard
+                [cbContent]="selectedToken.address"
+            >
+                {{ selectedToken.symbol }}
+            </span>
```

Comment:
maybe this could be a small component in the future, I am guessing this functionality is used a lot in the webui

Reply:
Yes we should refactor it